docs/canvas/canvas-function-reference.asciidoc

```diff
@@ -1,17 +1,2133 @@
+Set to `"auto"` to automatically set radius 
+
+Default: `"auto"`)
+
+|`hole`
+|`number`
+|Draw a hole in the pie, 0-100, as a percentage of the pie radius 
+
+Default: `0`
+
+|`labels`
+|`boolean`
+|Show pie labels 
+
+Default: `true`
+
+|`labelRadius`
+|`number`
+|The percentage of the area of container to use as a radius for the label circle 
+
+
+Default: `100`
+
+|`font`
+|`style`
+|The label font 
+
+Default: `{<<font_fn>>}`
+
+|`legend`
+|`string`, `boolean`
+|The legend position, `"nw"`, `"sw"`, `"ne"`, `"se"`, or `false` 
+
+Default: `false`
+
+|`tilt`
+|`number`
+|The percentage of tilt, where 1 is fully vertical and 0 is completely flat 
+
+Default: `1`
+|===
+
+*Returns:* `render`
+
+
+[float]
+[[plot_fn]]
+=== `plot`
+
+Configures a plot element.
+
+*Accepts:* `pointseries`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|`seriesStyle` *†*
+|`seriesStyle`
+|A style of a specific series
+
+|`defaultStyle`
+|`seriesStyle`
+|The default style to use for every series 
+
+
+Default: `{<<seriesStyle_fn>> points=5}`
+
+|`palette`
+|`palette`
+|A `palette` object for describing the colors to use on this plot 
+
+Default: `{<<palette_fn>>}`
+
+|`font`
+|`style`
+|The legend and tick mark fonts 
+
+Default: `{<<font_fn>>}`
+
+|`legend`
+|`string`, `boolean`
+|The legend position, `"nw"`, `"sw"`, `"ne"`, `"se"`, or `false` 
+
+Default: `"ne"`
+
+|`yaxis`
+|`boolean`, `axisConfig`
+|Axis configuration, or use `false` to disable 
+
+Default: `true`
+
+|`xaxis`
+|`boolean`, `axisConfig`
+|Axis configuration, or use `false` to disable 
+
+Default: `true`
+|===
+
+*Returns:* `render`
+
+
+[float]
+[[ply_fn]]
+=== `ply`
+
+Subdivides a `datatable` and passes the resulting tables into an expression, 
+then merges the output.
+
+*Accepts:* `datatable`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
 
-* <<canvas-common-functions>> can run anywhere
-* <<canvas-server-functions>> can only execute on the server
-* <<canvas-client-functions>> must execute in a browser
+|`by` *** *†*
+|`string`
+|The column to subdivide
+
+|`expression` *** *†*
+
+Alias: `fn`, `exp`, `function`
+|`datatable`
+|An expression to pass into each resulting data table.
+Expressions must return a `datatable`. Use `as` to turn literals into datatables.
+Multiple expressions must return the same number of rows. If you need to return 
+a differing row count, pipe into another instance of `ply`.				
+If multiple expressions return the same columns, the last one wins.
+|===
+
+*Returns:* `datatable`
+
+
+[float]
+[[pointseries_fn]]
+=== `pointseries`
+
+Turns a `datatable` into a point series model. Currently Canvas differentiates measure 
+from dimensions by looking for a <<canvas-tinymath-functions, Tiny Math function>>. If you enter a TinyMath 
+expression in your argument, Canvas treats that argument as a measure. Otherwise, it 
+is a dimension. Dimensions are combined to create unique keys. Measures are 
```

Comment:
How about:

`Measures are deduplicated by the keys that use the specified TinyMath function.`

Reply:
I think this makes more sense as is.

Reply:
I think you're right. I was probably overthinking this one.